### PR TITLE
Add esb-project-archetype

### PR DESCRIPTION
**Pull request by sajinieKavindya**

### CODE REVIEW

File: esb-project-archetype/pom.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (isudana) [inline esb-project-archetype/pom.xml]: Let's update the year to 2019

File: esb-project-archetype/src/main/resources/archetype-resources/pom.xml [hunk 2]
```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Copyright (c) 2016 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (isudana) [inline esb-project-archetype/src/main/resources/archetype-resources/pom.xml]: 2016 -> 2019

File: esb-project-archetype/src/main/resources/archetype-resources/pom.xml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+ ~ limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>${groupId}</groupId>
+    <artifactId>${artifactId}</artifactId>
+    <version>${version}</version>
+    <name>WSO2 EI - Synapse ESB project for integration solutions</name>
```
Comment (isudana) [inline esb-project-archetype/src/main/resources/archetype-resources/pom.xml]: Shall we use "WSO2 EI - ESB Project" as the name?

File: esb-project-archetype/pom.xml (hunk 1 above)
Comment (sajinieKavindya) [reply esb-project-archetype/pom.xml]: changed

**Pull request merged**